analytics/forecasting_v2/episode_lifecycle.py

```
@@ -2161,6 +2414,40 @@ def _assert_decision_matches_history(
+    Both are checked here, at the one boundary that turns a decision into
+    immutable history."""
+    risk = decision.planned_risk
+    if risk is None:
+        return
+    tick = read_creation_decision_tick_size(history)
+    if risk.decision_tick_size != tick:
+        raise V2EpisodeLifecycleError(
+            f"this decision's §18.1 decision_tick_size={risk.decision_tick_size} is not episode "
+            f"{history.episode_id!r}'s own persisted creation value {tick} -- §12.5a freezes the "
+            "creation-time tick grid for the episode's whole life, and today's instrument "
+            "metadata is never consulted")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)